dht/go.mod

```diff
@@ -36,3 +127,5 @@ require (
 //replace github.com/libp2p/go-libp2p-kad-dht => github.com/libp2p/go-libp2p-kad-dht v0.5.2-0.20200310202241-7ada018b2a13
 //replace github.com/libp2p/go-libp2p => github.com/libp2p/go-libp2p v0.6.1-0.20200310185355-89c193e0ca37
 //replace github.com/libp2p/go-libp2p-core => github.com/libp2p/go-libp2p-core v0.5.0
+
+replace github.com/multiformats/go-multiaddr-net => github.com/multiformats/go-multiaddr-net v0.1.5
```

Comment:
Why are there replace statements here?

Reply:
`go mod tidy` seems to auto-get go-multiaddr-net at v0.2.0, but this causes compile errors. v0.1.5 is the actual current released version, which works.

Without the replace statement, I get the errors:
```
# github.com/multiformats/go-multiaddr-net
../../go/pkg/mod/github.com/multiformats/go-multiaddr-net@v0.2.0/registry.go:25:26: undefined: manet.NetCodec
../../go/pkg/mod/github.com/multiformats/go-multiaddr-net@v0.2.0/registry.go:29:11: undefined: manet.RegisterNetCodec
```

Reply:
It's deprecated: https://github.com/multiformats/go-multiaddr-net.